src/coredump/coredump.c

```diff
@@ -1054,17 +1054,15 @@ static char* set_iovec_field_free(struct iovec iovec[27], size_t *n_iovec, const
 }
 
 static int gather_pid_metadata_and_process_special_crash(
-                const char *context[_CONTEXT_MAX],
+                char* context[_CONTEXT_MAX],
                 char **comm_fallback,
-                char **comm_ret,
```

Comment:
Another option would be to make it a normal structure and fill fields one by one. Don't know if that'd be better. I added a big comment for now.